src/seedu/addressbook/commands/ClockCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Prints date and time at exact moment when user enters "clock" command.
+ */
+public class ClockCommand extends Command {
+    public static final String COMMAND_WORD = "clock";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Prints date and time in format: day, date-month-year at hr:min:sec AM/PM timezone.\n"
+            + "Example: " + COMMAND_WORD;
+
+    @Override
```

Comment:
Missing header comment. All non-trivial methods should have java doc format header comments.